components/hal/esp32c6/include/hal/usb_serial_jtag_ll.h

```diff
@@ -168,6 +180,154 @@ static inline void usb_serial_jtag_ll_txfifo_flush(void)
     USB_SERIAL_JTAG.ep1_conf.wr_done=1;
 }
 
+/**
+ * @brief Enable USJ JTAG bridge
```

Comment:
What is USJ??

Reply:
I'll have a look on it to review

Reply:
Will you rename it to USB?

Reply:
Yes, I'll do a quick review on the files now and update

Reply:
It appears we have several references to USJ as the USB JTAG module (might be USb Jtag?), nine in this file alone and also present in 'usb_serial_jtag_types.h'. I'm not sure it is the case to change it.